rclcpp/test/test_parameter_client.cpp

```diff
@@ -69,7 +69,9 @@ TEST_F(TestParameterClient, async_construction_and_destruction) {
   {
     ASSERT_THROW(
     {
+      auto asynchronous_client =
```

Comment:
Please indent the following line since it is wrapped.

Same below.

Reply:
I get a uncrustify error when I do that. Any suggestion on how to resolve?
https://ci.ros2.org/job/ci_linux/9007/testReport/(root)/projectroot/uncrustify/

Reply:
That is weird. Plenty of existing code indents wrapped lines already.